includes/class-wc-gateway-ppec-checkout-handler.php

```diff
@@ -981,12 +983,14 @@ public function maybe_add_shipping_information( $packages ) {
 
 		$destination = $this->get_mapped_shipping_address( $checkout_details );
 
-		$packages[0]['destination']['country']   = $destination['country'];
-		$packages[0]['destination']['state']     = $destination['state'];
-		$packages[0]['destination']['postcode']  = $destination['postcode'];
-		$packages[0]['destination']['city']      = $destination['city'];
-		$packages[0]['destination']['address']   = $destination['address_1'];
-		$packages[0]['destination']['address_2'] = $destination['address_2'];
+		if ( ! empty( $destination ) ) {
+			$packages[0]['destination']['country']   = $destination['country'];
```

Comment:
Same concerns as above. Are we certain all these keys will be present? Would this be a good place to use wp_parse_args to make sure each key has a default?